test/functional/tests/common/lvm_common_utils.py

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright(c) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+
+import datetime
+
+from storage_devices.lvm import get_block_devices_list
+
+from api.cas import casadm
+from api.cas.cache_config import CacheMode
+from api.cas.init_config import InitConfig
+from test_tools import fs_utils
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import IoEngine, ReadWrite, VerifyMethod
+from test_utils.size import Size, Unit
+
+opencas_conf_path = "/etc/opencas/opencas.conf"
+
+
+def prepare_devices(cache_dev, core_dev, partitions_size: Size, cache_mode: CacheMode = None):
+    cache_dev.create_partitions([partitions_size])
+    core_dev.create_partitions([partitions_size])
+
+    cache = casadm.start_cache(cache_dev.partitions[0], cache_mode=cache_mode, force=True)
+    core = cache.add_core(core_dev.partitions[0])
+
+    return cache, core
```

Comment:
I think this is not a common method for lvm. It can be placed somewhere in the directory with tests, which use it